examples/notebooks/introduction_enlyze_python_sdk.ipynb

```diff
@@ -0,0 +1,2981 @@
+   "id": "a2f84359-c596-4d6c-b774-cf3f974dc72b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from enlyze.models import Site, Appliance, Variable"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca474120-c510-4f54-9345-f11808395324",
+   "metadata": {},
+   "source": [
+    "### Sites\n",
+    "\n",
+    "A list of all sites can be fetched by using the `get_sites` method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "e4b1d442-2521-49a8-8a67-468e8e96ea1e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[Site(_id=49, display_name='Platform', address='Maybachstrasse 156'),\n",
+       " Site(_id=32, display_name='Köln', address='Heliosstrasse 6a'),\n",
+       " Site(_id=35, display_name='Stuttgart', address='Mahlestrasse 67')]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sites = enlyze.get_sites()\n",
+    "sites"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bb02f9a1-3c75-46e0-b925-a55cc6827aeb",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Sidenote:</b> data-analysis in python 🫶 pandas\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0e44713b-9a76-45c6-abbb-813a6c781f93",
+   "metadata": {},
+   "source": [
+    "If you are new to processing data in python, you will inevitably come across pandas and pandas DataFrames. [Pandas](https://pandas.pydata.org/docs/index.html) is a great, open-source library providing high-performance, easy-to-use data structures and data analysis tools for the [Python](https://www.python.org/) programming language.\n",
+    "\n",
+    "Due to its many advantages for data processing and widespread use in the python ecosystem, it is already part of the enlyze package and is installed alongside as a requirement.\n",
+    "\n",
+    "Especially in [jupyter notebooks](https://jupyterlab.readthedocs.io/en/stable/getting_started/overview.html) it makes working with and visualizing data straightforward. You will notice that models (Site, Appliances, Variables etc.) have different attributes and are often returned in a list. For a few models and properties, printing the list in its raw form is still sufficient. But we soon reach the limits, and it becomes confusing and unmanageable. For this, we have this small utility function `models_to_dataframe` that takes a list of models and returns them as a DataFrame. This makes it a lot easier to explore data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "80593ce2-f90c-407f-92cc-12d04d8f75f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def models_to_dataframe(models: list):\n",
+    "    \"\"\"Convert a list of enlyze models to a dataframe\"\"\"\n",
+    "    return pd.DataFrame([m.__dict__ for m in models])\n",
+    "    "
```

Comment:
It's quite useful to me at least, but the production runs have already a similar behavior with the `.to_dataframe()` method. At the moment, this would be my preferred way.